app/models/embeddable.rb

```diff
@@ -79,6 +79,10 @@ def show_in_edit?
     true
   end
 
+  def is_hidden_plugin?
+    false
+  end
```

Comment:
`is_hidden_plugin?` is a bit confusing to me. As we're adding one more term on top of other plugin properties. I'd tend to be explicit and just check for `wrapping_plugin?` as then it's easier to understand this whole problem.
Using `hidden plugin` around, it makes me think that plugins can be somehow hidden.

But I'd leave it up to you whether to change that or not.